docs/tutorials/jupyter-index.md

```diff
@@ -0,0 +1,47 @@
+---
+id: tutorial-jupyter-index
+title: "Jupyter notebook tutorials"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+There are  Jupyter notebook-based tutorials that are available for you to try as you learn more about Druid. These tutorials provide snippets of Python code that you can use to run calls against the Druid API.
```

Comment:
```suggestion
There are Jupyter notebook-based tutorials that are available for you to try as you learn more about Druid. These tutorials provide snippets of Python code that you can use to run calls against the Druid API.
```

Reply:
Something I don't like about "There are". What about: You can try out the APIs using the notebooks... or something.

Reply:
```suggestion
You can try out the Druid APIs using the Jupyter notebook-based tutorials that are available. These tutorials provide snippets of Python code that you can use to run calls against the Druid API.
```